### PR TITLE
feat: add link to registry images

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
This adds a link on the image session setting to the container registry page in GitLab.

P.S. the screenshot may look slightly different than the preview because it includes the changes to the buttons from #1403 

image

/deploy renku=ui-design-2021

### CODE REVIEW

File: client/src/project/settings/ProjectSettings.present.js [hunk 1]
```diff
@@ -707,11 +708,20 @@ function SessionPinnedImage(props) {
     </Fragment>);
   }
 
+  const imagesUrl = `${repositoryUrl}/container_registry`;
+  const imagesLink = (<div>
+    <FormText>
+      You may want to check the {" "}
+      <ExternalLink role="link" title="available images for this project" url={imagesUrl} />.
```
Comment (ciyer) [inline client/src/project/settings/ProjectSettings.present.js:715]:
```suggestion
      A URL of a RenkuLab-compatible Docker image. For an image from this project, {" "}
       consult  <ExternalLink role="link" title="the list of images for this project" url={imagesUrl} />.
```

**Review by ciyer — CHANGES_REQUESTED** [review]
Thanks for putting this together so quickly! It makes it much easier to find an image.

While testing, I identified an an improvement that would be good to make; it can be done in this issue, if easy, otherwise, we should create a new issue:

If there are advanced settings set for this project, the advanced settings section should be open by default.

**Comment by lorenzo-cavazzi** [comment]
The suggested change for the images is actually very easy to implement (I initially implemented that, then removed it later).
Both points have been addressed in the latest commit.

**Review by ciyer — APPROVED** [review]
👍

**Pull request merged**